toonz/sources/toonz/mainwindow.cpp

```diff
@@ -1112,11 +1112,19 @@ void MainWindow::resetRoomsLayout() {
 }
 
 void MainWindow::maximizePanel() {
+  if (m_panelMaximized) {
+    m_activePanel->maximizeDock();
+    m_panelMaximized = false;
+    return;
+  }
```

Comment:
Confirmed that this PR fixes the bug.

However, for now this PR allows only one room to be maximized.
When you already have maximized panel in some room (say, room 1), to maximize some panel in another room (room 2) will need to hit the shortcut twice, one is for deactivating maximize in the room 1, another is for maximizing the panel in room 2.

How about having `m_panelMaximized` and `m_activePanel` in each room, instead of the `MainWindow`?